litex_boards/targets/cmod_a7.py

```diff
@@ -0,0 +1,124 @@
+"""
```

Comment:
Same as previous, can you remove this since the PR is related to Zedboard? (you could eventually do another PR for the CMOD A7, but we will need to uniformize it a bit to be similar to the others platforms/targets).